test/unit/spec/nbextensions/bulkImportCell/tabs/xsvGenerator-spec.js

```diff
@@ -248,21 +318,18 @@ define([
                         },
                     });
                     const xsvGen = new XSVGenerator({ model, typesToFiles, fileTypeMapping });
-                    spyOn(UI, 'showConfirmDialog').and.resolveTo(false);
-                    await xsvGen.run();
-                    expect(UI.showConfirmDialog).toHaveBeenCalledTimes(1);
-                    const calls = UI.showConfirmDialog.calls.allArgs()[0];
-                    expect(calls.length).toEqual(1);
-                    expect(calls[0].title).toEqual('Create CSV Template');
+                    const container = document.createElement('div');
+                    container.innerHTML = xsvGen.renderLayout();
```

Comment:
Alter this test to add the output to a container and test the contents of the container, rather than checking the arguments to the UI modal creation function.